template/tasks.py

```diff
@@ -227,6 +224,22 @@ def run_docs(ctx):
     ctx.run("poetry run mkdocs serve")
 
 
+@invoke.task
+def install(ctx, skip_install_playwright: bool = False):
+    """
+    Install system dependencies necessary for the {{ project_name }} project.
+
+    This task optionally skips the installation of Playwright dependencies,
+    which is useful for CI pipelines where Playwright is not needed,
+    thereby improving the build performance.
+    """
+    _title("Installing Dependencies")
+    ctx.run("poetry install")
+
+    if not skip_install_playwright:
+        _title("Installing Playwright Dependencies")
+        ctx.run("poetry run playwright install --with-deps")
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
**note (non-blocking):** Might be nice to also use this in the workflow for this repository itself. 

This repository current performs a poetry install and the runs the `check` invoke task. It might be nice use the `invoke install` task to do the install part before running `check`. Not blocking now but the reuse would be nice.‏

Reply:
Added [de97e35](https://github.com/ackama/django-template/pull/46/commits/de97e355d0b946b70e42026a325debb581caf6b6)